CondFormats/HcalObjects/interface/HcalQIEType.h

```diff
@@ -12,6 +12,7 @@ contains the QIE Typese + corresponding DetId
 class HcalQIEType
 {
  public:
+  enum {QIE8=0, QIE10=1, QIE11=2};
```

Comment:
Do you really need to have this enum in the condFormat? I guess this won't be used when the HcalQIEType payloads will be read from the DB... Maybe you could place this enum in a namespace and leave the CondFormat unchanged.